orbit/pkg/table/extension_darwin.go

```diff
@@ -83,14 +84,14 @@ func PlatformTables(opts PluginOpts) ([]osquery.OsqueryPlugin, error) {
 			},
 		),
 
-		filevault_status.TablePlugin(osqueryLogger), // table name is "filevault_status"
-		ioreg.TablePlugin(osqueryLogger),            // table name is "ioreg"
+		filevault_status.TablePlugin(log.Logger), // table name is "filevault_status"
```

Comment:
It took me a bit to understand the `log.Logger` the plugin functions here and in other files is the global logger. I wonder if it might be any clearer to assign it to a named variable somewhere in the table package (maybe in `extension`) where we could add some additional comments to help folks navigating the code in the future?

Reply:
Fair point! I thought about that as well; it can be kinda tricky unless you know that `log.Logger` is a global thing. I can bring back the old `osqueryLogger` variable if that works?

Reply:
Sounds good!

Reply:
@gillespi314 so I made this change, but now things aren't working correctly 😅 
The log output is now JSON, which isn't what's expected and doesn't work with `fleetd_logs` table. Should I just add a clarifying comment to the top of `extension.go` ? I can also dig deeper into this and find out why a variable assignment would bork the logger.

Reply:
I never had a good understanding of what the motivation was for the wrapper approach but I wonder if the shenanigans we're seeing here are related?

Reply:
Perhaps? I'm not 100% sure why the wrapper approach was taken either. Fwiw the issues seen that were the impetus for this ticket aren't the same as the ones I'm seeing after adding back `osqueryLogger`.